src/extensions/functions/checkRequirement.ts

```diff
@@ -22,6 +23,21 @@ module.exports = async (
     flatten
   )(requirement)
 
+  const taskWithFix = (checker: SolidarityChecker, rule: SolidarityRule, context: SolidarityRunContext) => async () => {
+    if (!shouldRunFix) return checker(rule, context)
+    try {
+      const result = await checker(rule, context)
+      return result
+    } catch (error) {
+      if (rule.fix) {
+        await context.system.run(rule.fix)
+        return checker(rule, context)
+      } else {
+        throw new Error('no fix found')
```

Comment:
could we be more specific here? For example "No fix script provided in .solidarity file"

Reply:
oooh, yis, small touchup @ryanlntn ?